packages/ember-routing/lib/system/router.js

```diff
@@ -129,6 +133,18 @@ var EmberRouter = EmberObject.extend(Evented, {
     return get(this, 'location').getURL();
   }),
 
+  _isModuleBasedResolver() {
```

Comment:
As an instance method on the router, `_isModuleBasedResolver()` might be named more clearly as `_hasModuleBasedResolver()` or even simply `_moduleBasedResolver()`.